x-pack/plugin/index-lifecycle/src/test/java/org/elasticsearch/xpack/indexlifecycle/IndexLifecycleInitialisationIT.java

```diff
@@ -204,4 +243,86 @@ private String getLocalNodeId(String name) {
         assertThat(nodeId, not(nullValue()));
         return nodeId;
     }
+
+    public static class TestILMPlugin extends Plugin {
+        public TestILMPlugin() {
+        }
+
+        public List<Setting<?>> getSettings() {
+            final Setting<Boolean> COMPLETE_SETTING = Setting.boolSetting("index.lifecycle.test.complete", false,
+                Setting.Property.Dynamic, Setting.Property.IndexScope);
+            return Collections.singletonList(COMPLETE_SETTING);
+        }
+        public List<NamedWriteableRegistry.Entry> getNamedWriteables() {
+            return Arrays.asList(new NamedWriteableRegistry.Entry(LifecycleType.class, LockableLifecycleType.TYPE,
+                    (in) -> LockableLifecycleType.INSTANCE),
+                new NamedWriteableRegistry.Entry(LifecycleAction.class, ObservableAction.NAME, ObservableAction::readObservableAction),
+                new NamedWriteableRegistry.Entry(ObservableClusterStateWaitStep.class, ObservableClusterStateWaitStep.NAME,
+                    ObservableClusterStateWaitStep::new));
+        }
+    }
+
+    public static class ObservableClusterStateWaitStep extends IndexLifecycleRunnerTests.MockClusterStateWaitStep
```

Comment:
Does this need to extend `IndexLifecycleRunnerTests.MockClusterStateWaitStep`? I don't think it uses anything from it?

Reply:
I will take another look. this may not be needed in the latest revision